html2docx/html2docx.py

```diff
@@ -132,6 +133,9 @@ def handle_starttag(self, tag: str, attrs: List[Tuple[str, Optional[str]]]) -> N
             self.init_run([])
         elif tag in ["b", "strong"]:
             self.init_run(["bold"])
+        elif tag == "blockquote":
+            self.style = "Quote"
+            # self.init_p(attrs)
```

Comment:
Is this Leftover?

```suggestion
```

Reply:
That's a discussion point. Do you think it is worth including this? It doesn't seem to me.